lib.php

```diff
@@ -239,9 +249,11 @@ function scormremote_pluginfile($course, $cm, $context, $filearea, $args, $force
             'layer3.js'
         );
 
+//            'jssource'         => $jssource . "?lms_origin={$origin}&student_id={$username}&student_name={$fullname}&mime='text/javascript'",
```

Comment:
this will all get moved around anyway but generally don't leave commented code in